packages/react-image/src/components/Image/useImage.ts

```diff
@@ -0,0 +1,38 @@
+import { getStyleFromPropsAndOptions } from '@fluentui/react-theme-provider';
+import { ComposePreparedOptions } from '@fluentui/react-compose';
```

Comment:
nit: Can we put these in alphabetical order?

```suggestion
import { ComposePreparedOptions } from '@fluentui/react-compose';
import { getStyleFromPropsAndOptions } from '@fluentui/react-theme-provider';
```

Reply:
Definitely, however I would like to avoid spending time on such things :)

@ecraig12345 Is it possible to enable [import/order](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md)? As I remember it contains even an autofix...

Reply:
Potentially. I like that the order is configurable. The default of `["builtin", "external", "parent", "sibling", "index"]` mostly seems reasonable to me, though we might want to use `pathGroups` to put monorepo things (`@fluentui/*`, `@uifabric/*`) between actual externals and relative paths.